src/components/Button.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+    <button :class="checkType(type)" @click="onClick" nativeType="submit">
```

Comment:
Add evaluation to _nativeType_ prop
```suggestion
  <button :class="checkType(type)" @click="onClick" :nativeType="nativeType">
```